src/scss/cdb-components/shapes/toogle-arrow.scss

```diff
@@ -7,17 +7,17 @@ Description
 <h2>16px</h2>
 <div style="padding: 20px;">
   <div class="CDB-Shape" style="background: rgba(0, 0, 0, .02);">
-    <div class="CDB-ArrowToogle"></div>
+    <div class="CDB-ArrowToggle"></div>
```

Comment:
In my opinion, if we keep using shapes, we should name them as:

```css
CDB-Shape-arrowToggle
CDB-Shape-magnify
```
cc @piensaenpixel @javierarce